app/views/locations/_form.html.erb

```diff
@@ -1,14 +1,8 @@
 <%= form_with(model: location, local: true) do |form| %>
   <% if location.errors.any? %>
-    <div id="error_explanation">
-      <h2><%= pluralize(location.errors.count, "error") %> prohibited this location from being saved:</h2>
-
-      <ul>
-        <% location.errors.full_messages.each do |message| %>
-          <li><%= message %></li>
-        <% end %>
-      </ul>
-    </div>
+      <% location.errors.full_messages.each do |message| %>
+        <%= javascript_tag "M.toast({html: '#{message}'}, 4000)" %>
```

Comment:
This looks like an injection vulnerability to me

Reply:
Injection how?

Reply:
See my other comment.  The exploit looks like either a `'` character or a `<script>` tag.